data/duke.txt

```diff
@@ -0,0 +1,4 @@
+T | 1 | test
+E | 1 | testing | 2022-08-31
+D | 1 | assignment | 2022-08-31
+T | 0 | testing
```

Comment:
Might want to edit the `.gitignore` file to ignore the save data files when pushing commits.

Reply:
I will implement it soon. Thanks